freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/FreeIpaClientFactory.java

```diff
@@ -142,17 +144,17 @@ private boolean canTryAnotherInstance(boolean lastInstance, IOException e) {
 
     public FreeIpaClient getFreeIpaClientForStack(Stack stack) throws FreeIpaClientException {
         LOGGER.debug("Creating FreeIpaClient for stack {}", stack.getResourceCrn());
-        return getFreeIpaClient(stack, false, Optional.empty());
+        return getFreeIpaClient(stack, false, false, Optional.empty());
     }
 
     public FreeIpaClient getFreeIpaClientForStack(Stack stack, String freeIpaFqdn) throws FreeIpaClientException {
         LOGGER.debug("Creating FreeIpaClient for stack {} for {}", stack.getResourceCrn(), freeIpaFqdn);
-        return getFreeIpaClient(stack, false, Optional.of(freeIpaFqdn));
+        return getFreeIpaClient(stack, false, false, Optional.of(freeIpaFqdn));
     }
 
-    public FreeIpaClient getFreeIpaClientForStackWithPing(Stack stack, String freeIpaFqdn) throws Exception {
+    public FreeIpaClient getFreeIpaClientForStackWithPingAndForceCheckUnreachableStack(Stack stack, String freeIpaFqdn) throws Exception {
```

Comment:
this method name get a little bit out of hand :)
could you make it shorter, like getFreeIpaClientForCheck or something similar